src/schema/nmdc.yaml

```diff
@@ -2401,6 +2404,23 @@ slots:
       Applies to a property that links a database object to a set of objects.
       This is necessary in a json document to provide context for a list, and to allow
       for a single json object that combines multiple object types
+
+  process_set:
+    mixins: object_set
+    range: PlannedProcess
```

Comment:
I was looking through the schema, and saw that `PlannedProcess` (and `MaterialEntity`) were both abstract classes.

I think the issue as to why JSON Schema validation isn't going through is because by asserting `range: PlannedProcess` you're saying that it's okay to create and initialize `process_set` with instances of `PlannedProcess` which we cannot do?